src/convertToObject.js

```diff
@@ -5,8 +5,27 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const cssObject = sourceString
+    .split(';')
+    .map((declaration) => declaration.trim())
+    .filter((declaration) => declaration !== '')
+    .map((declaration) => {
+      const propertyValuePair = declaration.split(':');
+
+      propertyValuePair[0] = propertyValuePair[0].trim();
+      propertyValuePair[1] = propertyValuePair[1].trim();
+
+      return propertyValuePair;
+    })
+    .reduce((accumulator, propertyValuePair) => {
+      accumulator[propertyValuePair[0]] = propertyValuePair[1];
+
+      return accumulator;
+    }, {});
+
+  return cssObject;
```

Comment:
The function `convertToObject` is correctly using string manipulation and array methods to convert a CSS string into an object. However, the variable `cssObject` could be named more descriptively to reflect its purpose, such as `stylesObject` or `cssPropertiesObject`.

Reply:
function name given from the task